matlab/msbuild/ice.proj

```diff
@@ -6,36 +6,30 @@
         <Configuration Condition="'$(Configuration)' == ''">Release</Configuration>
         <RootDir>$([System.IO.Path]::GetFullPath( $(MSBuildThisFileDirectory)..\msbuild ))</RootDir>
         <ToolboxDir>$(MSBuildThisFileDirectory)..\toolbox\build</ToolboxDir>
```

Comment:
This MSBuild project was trying to guess the version and the matlab home using the matlab executable from path. It was far too complicated.

I simplified to use a default and allow override with env variables. It was the usage of matlab here what was causing the CI issues.